app/assets/javascripts/task/notifications.js.coffee.erb

```diff
@@ -5,12 +5,15 @@ class @Notifications
     error: '<%= image_path "deploy_error.jpg" %>'
 
   @init: (outputStream) ->
-    outputStream.addEventListener('status', new this().statusUpdated)
+    if repoName = $('[data-repo-full-name]').attr('data-repo-full-name')
+      outputStream.addEventListener('status', new this(repoName).statusUpdated)
```

Comment:
`new this` looks odd to me but I guess that's the only way of supporting inheritance.